javascripts/background_scripts/posting_process.js

```diff
@@ -242,17 +242,5 @@ chrome.runtime.onMessage.addListener(
     }
   });
 
-// Respond to the request sent from posting_button.js with the value from localStorage["Options:DissableButton"]
-chrome.runtime.onMessage.addListener(
-  function(request, sender, sendResponse) {
```

Comment:
correct: it doesn't listen to option changed events. It make message calls directly to the options API to get the option value.